Sources/armory/renderpath/RenderPathDeferred.hx

```diff
@@ -53,7 +52,14 @@ class RenderPathDeferred {
 		#if rp_voxels
 		{
 			Inc.initGI();
+			#if arm_voxelgi_temporal
+			{
+				Inc.initGI("voxelsB");
+			}
+			#end
+			#if (rp_gi == "Voxel AO")
 			path.loadShader("shader_datas/deferred_light/deferred_light_VoxelAOvar");
+			#end
```

Comment:
Same question here, will this even work without your Voxel GI PR?

Reply:
yes